cdap-ui/app/cdap/styles/colors.scss

```diff
@@ -24,6 +24,7 @@ $blue-03: #0099ff;
 $blue-04: #58b7f6;
 $blue-05: #7cd2eb;
 $blue-06: #cae7ef;
+$blue-07: #3c4355;
```

Comment:
Should this color also be added to ThemeWrapper/colors.ts? If yes, maybe we should add a comment in colors.scss to do this for any new colors.

Reply:
Oops nvm, totally missed that you already did that